councilmatic_core/views.py

```diff
@@ -99,8 +99,6 @@ class IndexView(TemplateView):
     event_model = Event
 
     def get_context_data(self, **kwargs):
-        context = super(IndexView, self).get_context_data(**kwargs)
```

Comment:
we need this to get city-specific overrides of the homepage to work - this allows each city to have a custom context. can you add this back in? everything else looks like it's merely a style change, & looks fine